scoring/src/main/scala/ai/h2o/sparkling/ml/models/H2OMOJOPipelineModel.scala

```diff
@@ -118,17 +133,35 @@ class H2OMOJOPipelineModel(override val uid: String)
           }
       }
 
+      def mojoFrameToArray(mf : MojoFrame) = {
+        mf.getColumnNames.zipWithIndex.map {
```

Comment:
nit: another way to get the size could be to use the `getMeta()` and then `size()`